docs/tutorials/getting_started.md

```diff
@@ -0,0 +1,141 @@
+# Getting started
+
+## Prerequisites
+
+1. Python 3.11 or newer installed on your system.
+2. Access to a microgrid system supported by the `frequenz.sdk` or you can use
+   the sandbox.
+3. Basic knowledge of microgrid concepts.
+4. [Familiarity with Channels](https://frequenz-floss.github.io/frequenz-channels-python/latest/).
+5. [Install the Frequenz SDK](https://frequenz-floss.github.io/frequenz-sdk-python/latest/#installation)
```

Comment:
I was just researching on linking and we should probably use `[../index.md#installation]` so it works also when rendering locally.
```suggestion
5. [Install the Frequenz SDK](../index.md#installation)
```